src/FirstSteps/RPG/ItemModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using FirstSteps.RPG.Items;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FirstSteps.RPG
+{
+    public class ItemModel
+    {
+        public List<Item> ItemList { get; set; }
+    }
+}
```

Comment:
Why do I need an item model when they are in Inventory? I don't know how to use ItemModel.  Something like:
string serializeItem = Json.Serialize(hero.ToItemModel)? and same with serializeEquipment?

Reply:
The idea behind the model object is to have another class that represents data saved in a db/file. 
Now you have 'Hero.cs' with all behaviors and 'HeroModel.cs' that include only data representation. Equipment and items should have their own data representations like hero.